lib/src/seq.rs

```diff
@@ -50,11 +64,42 @@ macro_rules! declare_seq_with_contents_constraints_impl {
                 Self::from_slice(self, start_out, len)
             }
 
+            #[cfg_attr(feature="use_attributes", not_hacspec)]
+            pub fn native_slice(&self) -> &[T] {
+                &self.b
+            }
+
+            #[cfg_attr(feature="use_attributes", unsafe_hacspec)]
+            pub fn into_slice(mut self, start_out: usize, len: usize) -> Self {
```

Comment:
So now we have `Seq::slice` and `Seq::into_slice`, two functions that have the same signature ? Shouldn't they have more explicit names, or should we only keep one?

Reply:
Aaaah no I see `slice` borrows its argument while `into_slice` consumes it. Clever!